.github/actions/dbp-charts/verify-helm/helm_install.sh

```diff
@@ -1,41 +1,33 @@
-#!/bin/bash
+#!/bin/bash -e
 
-if [ -z "${ACS_VERSION}" ]; then
-  echo "ACS_VERSION variable is not set"
-  exit 2
-fi
-if [ -z "${COMMIT_MESSAGE}" ]; then
-  echo "COMMIT_MESSAGE variable is not set"
-  exit 2
-fi
-if [ -z "${ACM_CERTIFICATE}" ]; then
-  echo "ACM_CERTIFICATE variable is not set"
-  exit 2
-fi
-if [ -z "${AWS_SG}" ]; then
-  echo "AWS_SG variable is not set"
-  exit 2
-fi
-if [ -z "${GITHUB_RUN_NUMBER}" ]; then
-  echo "GITHUB_RUN_NUMBER variable is not set"
-  exit 2
-fi
-if [ -z "${DOMAIN}" ]; then
-  echo "DOMAIN variable is not set"
-  exit 2
-fi
-if [ -z "${BRANCH_NAME}" ]; then
-  echo "BRANCH_NAME variable is not set"
-  exit 2
-fi
+clean_up () {
+  if [[ "${COMMIT_MESSAGE}" != *"[keep env]"* ]]; then
+    echo "cleaning up..."
+    helm delete "${release_name_ingress}" "${release_name}" -n "${namespace}"
+    kubectl delete namespace "${namespace}" --grace-period=1
+  fi
+}
+trap clean_up EXIT
 
 GIT_DIFF="$(git diff origin/master --name-only .)"
-VALID_VERSION=$(echo "${ACS_VERSION}" | tr -d '.' | awk '{print tolower($0)}')
-namespace=$(echo "${BRANCH_NAME}" | cut -c1-28 | tr /_ - | tr -d [:punct:] | awk '{print tolower($0)}')-"${GITHUB_RUN_NUMBER}"-"${VALID_VERSION}"
-release_name_ingress=ing-"${GITHUB_RUN_NUMBER}"-"${VALID_VERSION}"
-release_name_acs=acs-"${GITHUB_RUN_NUMBER}"-"${VALID_VERSION}"
+if [[ "${ACS_VERSION}" == "none" ]]; then
+  echo "no acs version specified - setting parameters for aps"
+  namespace=$(echo "${BRANCH_NAME}" | cut -c1-28 | tr /_ - | tr -d '[:punct:]' | awk '{print tolower($0)}')"-${GITHUB_RUN_NUMBER}"
+  release_name_ingress=aps-ing-"${GITHUB_RUN_NUMBER}"
+  release_name=aps-"${GITHUB_RUN_NUMBER}"
+  PROJECT_NAME=alfresco-process-services
+  TEST_NEWMAN="false"
+else
+  echo "setting parameters for acs version ${ACS_VERSION}"
+  VALID_VERSION=$(echo "${ACS_VERSION}" | tr -d '.' | awk '{print tolower($0)}')
+  namespace=$(echo "${BRANCH_NAME}" | cut -c1-28 | tr /_ - | tr -d [:punct:] | awk '{print tolower($0)}')-"${GITHUB_RUN_NUMBER}"-"${VALID_VERSION}"
+  release_name_ingress=ing-"${GITHUB_RUN_NUMBER}"-"${VALID_VERSION}"
+  release_name=acs-"${GITHUB_RUN_NUMBER}"-"${VALID_VERSION}"
+  PROJECT_NAME=alfresco-content-services
+  TEST_NEWMAN="true"
+fi
```

Comment:
in the end I think we just want to generate an unique project name, so maybe just provide a prefix that needs to be provided as an input for the action and use that here (without any ifs)